src/test/java/Pages/HomePage.java

```diff
@@ -14,8 +15,25 @@ public HomePage(WebDriver givenDriver) {
 
     By userAvatarIcon = By.cssSelector("img.avatar");
 
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By playlistNameField = By.cssSelector("[name='name']");
+    By renamePlaylistSuccessMsg = By.cssSelector("div.success.show");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```